### ISSUE TITLE
Multiple titles have issues with ordering and visibility.

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/adventist_knapsack#805]

**Comment by ShanaLMoore** [comment]
@KatharineV to confirm, your example links to your prod site, so this isn't a valkyrie related bug? You're requesting a fix on prod?

**Comment by KatharineV** [comment]
Yes that’s right. That’s why I didn’t add the Valkyrie tag. This is a post Knapsack bug but I didn’t catch it until now, so I’m just treating it like a normal issue report.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by jillpe** [comment]
SoftServ QA: ✅ 

![Image](https://github.com/user-attachments/assets/c34f117d-5f08-45b0-8eb5-05dbb804b552)

![Image](https://github.com/user-attachments/assets/b7151146-5859-4902-b391-4c2072165f94)

**Comment by KatharineV** [comment]
I created a thesis work with two titles. I saved, edited, resaved, and the titles remained in the order I entered them. This is good behavior that we expect to see.
https://adl.s2.adventistdigitallibrary.org/concern/theses/wdf_0003_031__the_contributions_of_twelve_men_who_graduated_from_oakwood_co

Collections continue to hide the second title, but that is a pre-Valkyrie issue that we will ignore. Thank you for fixing the reordering issue as it affected other kinds of works.

**Comment by KatharineV** [comment]
Tested in Demo.b2 and behavior is as expected. Note that the second title does NOT appear on the collection landing page. This matches expectations, although it may surprise future testers.

https://demo.b2.adventistdigitallibrary.org/collections/d3317a72-e9f6-4d9f-91ee-093ca47f5ec8
